src/lib/core/datetime/public-api.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+export {
```

Comment:
you can mark all of this as deprecated, people should start importing it from cdk instead